idea_town/frontend/static-src/app/views/base-view.js

```diff
@@ -1,16 +1,36 @@
+import app from 'ampersand-app';
 import AmpersandView from 'ampersand-view';
 import flatten from 'lodash/array/flatten';
 import invoke from 'lodash/collection/invoke';
+import localLinks from 'local-links';
 import mustache from 'mustache';
 // BaseView just abstracts out stuff we seem to use in all the views
 
 export default AmpersandView.extend({
   // override _template with a mustache template
   _template: '',
+
   template(ctx) {
     return mustache.render(this._template, ctx);
   },
 
+  events: {
+    'click a[href]': 'handleLinkClick'
+  },
+
+  // Handles all `<a>` clicks in the app not handled
+  // by another view. This lets us determine if this is
+  // a click that should be handled internally by the app.
```

Comment:
Catchall _was_ the intent. As with the handler in `main.js` this was another thing I pulled from the ampersand generated boilerplate project. Both of these things were my way of creating a more solid foundation(trying to follow by example the ampersand way of doing things) in order to debug easier. Will reassess now that I understand the issues a bit better :)